util/weekly_build/06_AppTests.sh

```diff
@@ -2,8 +2,19 @@
 
 set -ex
 
-if [ -z $SETUPDONE ]; then . ShellSetup.sh $* ; fi
+cd $RUNDIR/$RUNID
+
+set +x
+. setup.sh
+set -x
+
+for compiler in $COMPILERS; do
+  cd $RUNDIR/$RUNID/envs/build-${compiler/@/-}
+  spack env activate .
+  spack module lmod refresh -y
+  spack stack setup-meta-modules
+done
 
 if [ "$TEST_UFSWM" == ON ]; then
-  ./apptests/test_ufswm.sh
+  ./apptests/test_ufswm.sh $*
```

Comment:
Would it be possible to configure the rt.sh arguments so they can be set by an environment variable, kind of like you did with UFSWM_BRANCH and UFSWM_URL? I think that will make both site-specific configuration and debugging/testing easier.

Reply:
Site-specific config (for **orion**) added, generalizing `rt.sh` arguments